modules/virt-editing-vm-web.adoc

```diff
@@ -14,31 +14,40 @@
 ifeval::["{context}" == "virt-editing-vm-template"]
 :virt-vm-template:
 :object: virtual machine template
-:object-gui: Virtual Machines
-:object-vm-overview: VM Template Details
+:object-gui: Templates
+:object-vm-overview: Virtual Machine Overview
+:object-vm-details: VM Template Details
```

Comment:
Is adding this new conditional variable necessary?

Reply:
This conditional is being used in the text:
= Editing a {object} in the web console

Edit select values of a {object} that displays in the *{object-vm-details}* screen of the web console by clicking on the pencil icon next to the relevant field. Other values can be edited using the CLI.

Reply:
@lmandavi  That step seems to have been rewritten and the `object-vm-detail` is no longer used in any of these modules. If that remains the case then it should be removed from the variable set/unset sections for each of these modules. 
However, I believe @sjhala-ccs is saying in comments further up that you could use the variable in those steps to improve that step in the procedure. In which case, please keep the setting/unsetting of that `object-vm-detail` variable in the relevant modules.